generate_rss.py

```diff
@@ -128,8 +128,8 @@ def main():
 
     # Add metadata
     fg.title("Newest Python PEPs")
-    fg.link(href="https://www.python.org/dev/peps")
-    fg.link(href="https://www.python.org/dev/peps/peps.rss", rel="self")
+    fg.link(href="https://peps.python.org")
+    fg.link(href="https://peps.python.org/peps.rss", rel="self")
```

Comment:
Should there already be a feed at https://peps.python.org/peps.rss?

The old https://www.python.org/dev/peps/peps.rss redirect there too.

Reply:
Fixed (36461620)

A